crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -196,6 +384,20 @@ fn removed_method(checker: &mut Checker, expr: &Expr) {
             )),
             &_ => None,
         },
+        ["airflow", "datasets", ..] | ["airflow", "Dataset"] => match attr.as_str() {
+            "iter_datasets" => Some(Replacement::Name("iter_assets")),
+            "iter_dataset_aliases" => Some(Replacement::Name("iter_asset_aliases")),
+            &_ => None,
+        },
+        _ if is_airflow_secret_backend(qualname.segments()) => match attr.as_str() {
+            "get_conn_uri" => Some(Replacement::Name("get_conn_value")),
+            "get_connections" => Some(Replacement::Name("get_connection")),
+            &_ => None,
+        },
+        _ if is_airflow_hook(&qualname) => match attr.as_str() {
+            "get_connections" => Some(Replacement::Name("get_connection")),
+            &_ => None,
+        },
         _ => None,
```

Comment:
nit: I think we should just combine this in the catch-all like:

```rs
if is_airflow_secret_backend(qualname.segments()) {
	// ...
} else if is_airflow_hook(&qualname) {
	// ...
}
```

Assuming that these are mutually exclusive which I think it is as it's in separate match arms.

Reply:
Sure! Updated!